loadgen/demos/demo.py

```diff
@@ -0,0 +1,41 @@
+"""
+Python demo showing how to use the MLPerf Inference load generator bindings.
+"""
+
+from __future__ import print_function
+from absl import app
+import mlpi_loadgen
+import threading
+import time
+
+def LoadSamplesToRam(query_samples):
+  return
+
+def UnloadSamplesFromRam(query_samples):
+  return
+
+def ProcessQueryAsync(query_id, response_count):
+  time.sleep(.01)
+  r1 = mlpi_loadgen.QuerySampleResponse(0, 0);
+  r2 = mlpi_loadgen.QuerySampleResponse(0, 0);
+  r3 = mlpi_loadgen.QuerySampleResponse(0, 0);
+  r4 = mlpi_loadgen.QuerySampleResponse(0, 0);
+  responses = [r1, r2, r3, r4]
+  mlpi_loadgen.QueryComplete(query_id, responses)
+
+def IssueQuery(query_id, query_samples):
+  print(query_samples)
+  threading.Thread(target=ProcessQueryAsync, args=(query_id, len(query_samples))).start()
+
+def main(argv):
+  sut_name = "DemoSUT"
+  sut = mlpi_loadgen.ConstructSUT(sut_name, IssueQuery)
+  qsl_name = "DemoQSL"
+  qsl = mlpi_loadgen.ConstructQSL(qsl_name, 1024, 128, LoadSamplesToRam, UnloadSamplesFromRam)
+  command_line = "--mlperf_scenario edge"
+  mlpi_loadgen.StartTest(sut, qsl, command_line)
```

Comment:
I think we'd not want a library parse the apps command line - how about a simple string called 'mode' or if we need to  have more arguments a struct options { ... }

Reply:
Added python bindings for the TestSettings struct.